content/rest/messages.textile

```diff
@@ -157,6 +157,12 @@ bc[swift]. channel.publish("event", data: "This is my payload")
 
 bc[go]. channel.Publish("event", "This is my payload")
 
+h4(#idempotent). Enabling idempotent publishing
+
+It is possible for a client publishing through REST to not receive an acknowledgement of receipt from Ably for numerous reasons, which can lead to the client re-publishing a message. If you wish to ensure that Ably can recognise these re-sent messages as duplicates, you should "enable idempotent publishing":/rest/usage#client-options. This attaches an identifier to each message sent from the client, enabling Ably to detect duplicates of the same message and ignore them.
```

Comment:
"for numerous reasons" sounds like our system is faulty.  We need to be more explicit, "for numerous reasons such as network failures outside of our control. This can lead to the client automatically re-publishing the message ., which is a [feature of our SDKs to help route around network failures](https://ably.freshdesk.com/support/solutions/articles/3000044636-routing-around-network-and-dns-issues). We recommend that you "enable idempotent publishing":/rest/usage#client-options in our REST SDKs if you want to avoid these retry attempts potentially resulting in duplicate messages.  When enabled, the REST SDK automatically assigns a unique ID to each message ensuring subsequent retries cannot result in duplicates."

Perhaps add an intro line: "Idempotency ensures that multiple publishes of the same message cannot result in duplicates messages. [Find out more about how what idempotency is, and how we provide idempotency in our REST operations](https://www.ably.io/concepts/idempotency)."